src/server/scripts/Northrend/FrozenHalls/PitOfSaron/instance_pit_of_saron.cpp

```diff
@@ -157,7 +157,7 @@ class instance_pit_of_saron : public InstanceMapScript
                     case DATA_GARFROST:
                         if (state == DONE)
                         {
-                            if (Creature* summoner = instance->GetCreature(_garfrostGUID))
```

Comment:
Is this check needed for anything? I suspect this is a leftover from Garfrost originally summoned the spawns.

Same for Tyrannus.

Reply:
You seem to be right. Commit for reference: https://github.com/TrinityCore/TrinityCore/commit/ef694cc251844d34a64ae03f79d28a5c76719f6b

@jackpoz any insight?

Reply:
If the summoner npc is not there (whatever is the reason), it makes sense not to spawn the summons. I didn't want to change that logic so I left the null check.

Reply:
Alright. I'll leave it as it is